app/controllers/course/course_users_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class Course::CourseUsersController < Course::ComponentController
+  load_and_authorize_resource :course_user, through: :course
+  helper 'course/achievements'
```

Comment:
I'd prefer `Course::AchievementsHelper.name.sub(/Helper$/, '')` because that indicates a reference that your IDE can resolve.